tools/http_server.py

```diff
@@ -17,8 +17,17 @@
 ANOTHER_REDIRECT_PORT = 4547
 DOUBLE_REDIRECTS_PORT = 4548
 
+QUIET = '-vv' not in sys.argv and sys.argv.count('-v') < 2
```

Comment:
Can `QUIET` be taken out from arg parser?

Reply:
I think not since it's imported by benchmark.py and maybe others, which have different args than test.py...